### PR TITLE
Key self-hosted config setting recommendations for large file imports

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/6379

### CODE REVIEW

File: source/manage/cloud-data-export.rst [hunk 1]
```diff
@@ -89,7 +94,28 @@ This will show all of the exports on the server, so be sure to download the late
 Upload the export to the new server
 -----------------------------------
 
-Finally, it's time to take our export from the source server and use it as an import into the destination server. First, log into the destination server using ``mmctl`` the same way you logged into the source server:
+Finally, it's time to take our export from the source server and use it as an import into the destination server. If you have a file import larger than <x> and plan to import to a self-hosted Mattermost instance, first, review and modify the following self-hosted Mattermost configuration settings, where applicable, to ensure a smooth and successful import:
```
[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/manage/cloud-data-export.rst:97]: After chatting with Stu, we've aligned on:
- rewording this paragraph text to focus less on a specific value recommendation and more on awareness that the max file size config setting needs to be set to a value compatible with the customer's need to import. In addition, update recommendation to reduce the max file size value back to a reasonable value for daily use for users once migration is complete.
- updating read timeout/write timeout recommendations to focus on the need to tune these values depending on import file speed and network path to allow the file to upload without timeouts, and then we recommend reverting the values back to previous values once the import is successful and migration is complete. (AKA customer won't want these values set very high all the time).

**Comment by cwarnermm** [comment]
@nab-77 @sadohert - This docs PR is ready for your final review.

**Review by sadohert — APPROVED** [review]
LGTM... thanks!

**Review by nab-77 — APPROVED** [review]
Good guidance here.

**Pull request merged**